samples/Common.c

```diff
@@ -1411,6 +1414,10 @@ STATUS signalingMessageReceived(UINT64 customData, PReceivedSignalingMessage pRe
         freeMessageQueue(pPendingMessageQueue);
     }
 
+    if (freeStreamingSession && pSampleStreamingSession != NULL) {
```

Comment:
If handleOffer fails, `freeStreamingSession` would be false which means this still wouldnt be freed right? Also, shouldnt we free sample streaming session in case of any call failure?

Reply:
+1

Reply:
Addressed this by moving the bool right after creating the streaming session